editor/src/core/workers/parser-printer/ts-creator/transformer.ts

```diff
@@ -1609,15 +1606,29 @@ function produceNodeFromNode(
 }
 
 export function transformSourceFile(sourceText: string, sourceFile: SourceFile): SourceFile {
-  const updatedStatements = sourceFile.statements.map((statement) => {
-    const lastToken = sourceFile.getLastToken(sourceFile)
-    let additionalTrailingComments: Array<ParserPrinterComment> = []
-    if (lastToken != null && lastToken.kind === SyntaxKind.EndOfFileToken) {
-      // A sneaky way to take comments attached to the EndOfFileToken and put them on the node
-      // that is created. As this code can be run for segments of an actual file, it doesn't
-      // even make sense for there to be a parallel of the EndOfFileToken in what we're handling.
-      const lastTokenComments = getComments(sourceText, lastToken)
-      additionalTrailingComments = lastTokenComments.leadingComments
+  const fileLastToken = sourceFile.getLastToken(sourceFile)
```

Comment:
Does the below fix mean we can now remove this unsupported case? https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/workers/parser-printer/parser-printer-comments.spec.ts#L103

Reply:
Nope, as that handling is for arbitrary blocks, potentially it could be co-opted/refactored to be used to fix that though.

Reply:
Got ya :+1: